strings/strings.c

```diff
@@ -20,7 +21,18 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int length = strlen(s);
+    int end = length - 1;
+    int begin = 0;
+
+    for (begin = 0; begin < length; begin++)
```

Comment:
let me know if the tests are working for you, hopefully your code is passing just fine

Reply:
Tests are passing for me, are they not for you?